src/renderer/modals/login/loginModal.tsx

```diff
@@ -80,11 +94,14 @@ export default class LoginModal extends React.PureComponent {
             theServer = `The proxy ${this.state.authInfo.host}:${this.state.authInfo.port}`;
         } else {
             const tmpURL = urlUtils.parseURL(this.state.request.url);
-            theServer = `The server ${tmpURL.protocol}//${tmpURL.host}`;
+            theServer = `The server ${tmpURL?.protocol}//${tmpURL?.host}`;
         }
         const message = `${theServer} requires a username and password.`;
         return (
-            <Modal show={Boolean(this.state.request && this.state.authInfo)}>
+            <Modal
+                show={Boolean(this.state.request && this.state.authInfo)}
+                onHide={() => {}}
```

Comment:
I know we do this elsewhere, but this will cause the Modal to re-render whenever the LoginModal does.

Reply:
Is this in regards to passing in the state or the blank function?
For the blank function, that's a typing issue with `react-bootstrap` v0.x that will be resolved here: https://github.com/mattermost/desktop/pull/1639
For the state, I believe it's only ever set once in an above function so the most this will ever re-render should be once.